modules/ztp-configuring-pgt-image-registry.adoc

```diff
@@ -1,101 +1,175 @@
+====
+
+. Commit the `PolicyGenTemplate` change in Git, and then push to the Git repository being monitored by the GitOps ZTP ArgoCD application.
 
 .Verification
 
-. Check that the `Config` CRD of the group `imageregistry.operator.openshift.io` instance is not reporting errors. Run the following command:
+Use the following steps to troubleshoot errors with the local image registry on the managed clusters:
 
-. Check that the `PersistentVolumeClaim` on the managed cluster is populated with data. Run the following command:
+* Verify successful login to the registry while logged in to the managed cluster. Run the following commands:
 
-. Check that the `registry*` pod is up correctly located under the `openshift-image-registry` namespace.
+.. Export the managed cluster name:
++
+[source,terminal]
+----
+$ cluster=<managed_cluster_name>
+----
+
+.. Get the managed cluster `kubeconfig` details:
++
+[source,terminal]
+----
+$ oc get secret -n $cluster $cluster-admin-password -o jsonpath='{.data.password}' | base64 -d > kubeadmin-password-$cluster
+----
 
-. Verify successful login to the registry with `podman`:
+.. Download and export the cluster `kubeconfig`:
 +
 [source,terminal]
 ----
-$ oc login -u kubeadmin -p <password_from_install_log> https://api-int.<cluster_name>.<base_domain>:6443
+$ oc get secret -n $cluster $cluster-admin-kubeconfig -o jsonpath='{.data.kubeconfig}' | base64 -d > kubeconfig-$cluster
 ----
 +
 [source,terminal]
 ----
-$ podman login -u kubeadmin -p $(oc whoami -t) image-registry.openshift-image-registry.svc:5000
+$ export KUBECONFIG=./kubeconfig-$cluster
 ----
 
-. Check for disk partitioning using `lsblk` to list your blocks:
+.. Verify access to the image registry from the managed cluster. See "Accessing the registry".
+
+* Check that the `Config` CRD in the `imageregistry.operator.openshift.io` group instance is not reporting errors. Run the following command while logged in to the managed cluster:
 +
 [source,terminal]
 ----
-$ oc debug node/sno-1.example.com
+$ oc get image.config.openshift.io cluster -o yaml
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Image
+metadata:
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/create-only: "true"
+  creationTimestamp: "2021-10-08T19:02:39Z"
+  generation: 5
+  name: cluster
+  resourceVersion: "688678648"
+  uid: 0406521b-39c0-4cda-ba75-873697da75a4
+spec:
+  additionalTrustedCA:
+    name: acm-ice
+----
+
+* Check that the `PersistentVolumeClaim` on the managed cluster is populated with data. Run the following command while logged in to the managed cluster:
++
+[source,terminal]
+----
+$ oc get pv image-registry-sc
+----
+
+* Check that the `registry*` pod is running and is located under the `openshift-image-registry` namespace.
```

Comment:
I noticed you're using unordered steps, but I think this could work just fine as numbered steps with `.` instead of `*`.